futures-channel/src/lib.rs

```diff
@@ -1,6 +1,10 @@
-//! Asynchronous channels
+//! Asynchronous channels.
 //!
-//! This crate provides channels which can be used to communicate between futures.
+//! This crate provides channels that can be used to communicate between
+//! asynchronous tasks.
+//!
+//! See [Asynchronous Programming in Rust](https://aturon.github.io/apr/) for an
+//! overview.
```

Comment:
This looks like it will be awesome as more chapters are filled in. Though, at the moment, should it actually be linked here? I noticed a lot of things that are different in APR versus what is in futures 0.2, which may cause more confusion than clarity.

Reply:
It'll be updated before 0.2 is published.

Reply:
Oh cool, looking forward to it!